common/scripts/kind_provisioner.sh

```diff
@@ -190,7 +190,16 @@ function setup_kind_cluster() {
     kind create cluster --name="${NAME}" -v4 --retain --image "${IMAGE}" ${KIND_WAIT_FLAG:+"$KIND_WAIT_FLAG"} --config -); then
     echo "Could not setup KinD environment. Something wrong with KinD setup. Exporting logs."
     return 9
+    # kubectl config set clusters.kind-istio-testing.server https://istio-testing-control-plane:6443
   fi
+
+  if [[ -n "${DEVCONTAINER}" ]]; then
```

Comment:
```suggestion
   if [[ -n "${DEVCONTAINER:-}" ]]; then
```

Reply:
https://github.com/istio/common-files/pull/1142

Reply:
Ah okay, looks like that should fix it, saw change was originally introduced in https://github.com/istio/common-files/pull/1140

Reply:
Fix was merged by automator in #54799, will try rebasing to see if I get stuck by branch protection again.